tests/foreman/ui_airgun/test_contenthost.py

```diff
@@ -213,6 +213,52 @@ def test_positive_end_to_end(session, repos_collection, vm):
             assert not session.contenthost.search(vm.hostname)
 
 
+@upgrade
+@tier3
+def test_positive_end_to_end_bulk_update(session, vm):
+    """Create VM, set up VM as host, register it as a content host,
+    read content host details, install a package ( e.g. walrus-0.71) and
+    use bulk action (Update All Packages) to update the package by name
+    to a later version.
+
+    :id: d460ba30-82c7-11e9-9af5-54ee754f2151
+
+    :expectedresults: package installation and update to a later version
+        are successful.
+
+    :BZ: 1712069
+
+    :CaseLevel: System
+    """
+    hc_name = gen_string('alpha')
+    description = gen_string('alpha')
+    result = vm.run('yum -y install {0}'.format(FAKE_1_CUSTOM_PACKAGE))
+    assert result.return_code == 0
+    with session:
+        # Ensure content host is searchable
+        assert session.contenthost.search(vm.hostname)[0]['Name'] == vm.hostname
+        # Update package using bulk action
+        # use the Host Collection view to access Update Packages dialogue
+        session.hostcollection.create({
+            'name': hc_name,
+            'unlimited_hosts': False,
+            'max_hosts': 2,
+            'description': description
+        })
+        session.hostcollection.associate_host(hc_name, vm.hostname)
+        # Update the package by name
+        session.hostcollection.manage_packages(
```

Comment:
yes, that is invalid as main goal of the test case is to perform that operation through content host

Reply:
@oshtaier I did ask the SME, developer assigned to the bug[1], and as long as we see "Actions::Katello::Host::Package::Update action is planned in the Monitor->Tasks entry " then we are checking the code path where the problem occurred.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1712069#c11